releases/release-6.6.0.md

```diff
@@ -0,0 +1,440 @@
+---
+title: TiDB 6.6.0 Release Notes
+---
+
+# TiDB 6.6.0 Release Notes
+
+Release date: xx, 2023
+
+TiDB version: 6.6.0-DMR
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.6/quick-start-with-tidb) | [Installation package](https://cn.pingcap.com/product-community/)
+
+In v6.6.0-DMR, the key new features and improvements are as follows:
+
+- MySQL 8.0 兼容的多值索引 (Multi-Valued Index) (实验特性)
+- 基于资源组的资源管控 (实验特性)
+- 悲观锁队列的稳定唤醒模型
+- 数据请求的批量聚合
+
+## New features
+
+### SQL
+
+* Support dynamically managing the resource usage of DDL operations (experimental) [#38025](https://github.com/pingcap/tidb/issues/38025)  @[hawkingrei](https://github.com/hawkingrei) **tw@ran-huang**
+
+    TiDB v6.6.0 introduces resource management for DDL operations to reduce the impact of DDL changes on online applications by automatically controlling the CPU usage of these operations. This feature is effective only after the [DDL distributed parallel execution framework](/system-variables.md#tidb_ddl_distribute_reorg-new-in-v660) is enabled.
+
+* Support the MySQL-compatible foreign key constraints [#18209](https://github.com/pingcap/tidb/issues/18209) @[crazycs520](https://github.com/crazycs520) **tw@Oreoxmt**
+
+    TiDB v6.6.0 introduces the foreign key constraints feature, which is compatible with MySQL. This feature supports referencing within a table or between tables, constraints validation, and cascade operations. This feature helps to maintain data consistency, improve data quality, and facilitate data modeling.
+
+    For more information, see [documentation](/foreign-key.md).
+
+* Support rolling back DDL operations via the `FLASHBACK CLUSTER TO TIMESTAMP` statement [#14088](https://github.com/tikv/tikv/pull/14088) @[Defined2014](https://github.com/Defined2014) @[JmPotato](https://github.com/JmPotato) **tw@ran-huang**
+
+    The [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) statement supports restoring the entire cluster to a specified point in time within the Garbage Collection (GC) lifetime. In TiDB v6.6.0, this feature adds support for rolling back DDL operations. This can be used to quickly undo a DML or DDL misoperation on a cluster, roll back a cluster within minutes, and roll back a cluster multiple times on the timeline to determine when specific data changes occurred.
```

Comment:
Does it only rollback the schema or the schema and data? This isn't really clear from the text here.

Reply:
It will rollback schema and data. The text said `restoring the entire cluster to a specified point`. Do you have comments how to change it will be better here?

maybe change `a DML or DDL misoperation` to `DML and DDL misoperations` will be better?

Reply:
```suggestion
    The [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) statement supports restoring the entire cluster to a specified point in time within the Garbage Collection (GC) lifetime. In TiDB v6.6.0, this feature adds support for rolling back any DDL or DML operations. This can be used to quickly undo a DML or DDL misoperation on a cluster, roll back a cluster within minutes, and roll back a cluster multiple times on the timeline to determine when specific data changes occurred.
```

Maybe something like this?

Reply:
This release adds support for DDL. `flashback` DML has been supported in v6.5.0.

And not all DDLs are supported, some DDLs that modify etcd will not be supported.

Reply:
Maybe change to `In TiDB v6.6.0, this feature adds support for rolling back DDL operations. Now this can be used to quickly undo most DML and DDL misoperations on a cluster,` is better? @dveeden